target_blob_storage/__init__.py

```diff
@@ -45,13 +46,30 @@ def parse_args():
 def upload(args):
     logger.info(f"Exporting data...")
     config = args.config
-    container_name = config['container']
-    target_path = config['path_prefix']
-    local_path = config['input_path']
-    connect_string = config['connect_string']
 
-    # Upload all data in input_path to Azure Blob Storage
-    blob_service_client = BlobServiceClient.from_connection_string(connect_string)
+    connect_string = config['connect_string']
```

Comment:
This will fail if `connect_string` is not in the `config` dict. Should be `config.get('connect_string')`